js1.js

```diff
@@ -0,0 +1,12 @@
+"use script"
+
+let a=5;
+let b=6;
+let c=9;
+let P=a+b+c; //Search perimeter of triangle
+let p=P/2; //Search half-perimeter of triangle
+let S=(p*(p-a)*(p-b)*(p-c))**(1/2); //Search Square of triangle by the Herons formula
+S=S.toFixed(2); //Rounding to the hundredth
```

Comment:
you should not write comments. You can write code which will be more easy to understand.  Use naming of variables better